models/agent_model.go

```diff
@@ -262,13 +276,7 @@ func (s *Agent) DSN(service *Service, dialTimeout time.Duration, database string
 		cfg.DBName = database
 		cfg.Params = make(map[string]string)
 		if s.TLS {
-			// TODO: how certs and other parameters are going to be specified? We need to implement calling RegisterTLSConfig
-			// See https://godoc.org/github.com/go-sql-driver/mysql#RegisterTLSConfig
-			if s.TLSSkipVerify {
-				cfg.Params["tls"] = "skip-verify"
-			} else {
-				cfg.Params["tls"] = "true"
-			}
+			cfg.Params["tls"] = "custom"
```

Comment:
I would say that custom should be if custom certificates are not empty. if skip-verify is passed it should be `skip-verify` and if just `tls` is passed then it should be `true`.

Reply:
let's include it to the new task.